middleware/limiter/config.go

```diff
@@ -102,5 +109,10 @@ func configDefault(config ...Config) Config {
 	if cfg.LimiterMiddleware == nil {
 		cfg.LimiterMiddleware = ConfigDefault.LimiterMiddleware
 	}
+	if cfg.MaxCalculator == nil {
+		cfg.MaxCalculator = func(_ fiber.Ctx) int {
+			return cfg.Max
```

Comment:
If we don't use cfg.Max in the middleware even though cfg.MaxCalculator is not set, i think there is no need to keep a config property called Max. 

Default MaxCalculator simply do the same thing as Max:
```go
func(_ fiber.Ctx) int {
	return 5
}
```

Reply:
But what if someone wants to only set the number and nothing else?

I don't think it would be a nice API if we just left the function there, I think it's a nice option to leave the cfg.Max key. I just have a default function for MaxGenerator to avoid doing a lot of ifs inside the middleware itself, so it only calls the function to generate the max and uses it. I can go back to use cfg.Max if necessary but I'm sure that would probably affect the benchmarks a lot more then just having the default function.

Reply:
Yes you are right then, lets keep it

Reply:
By the way, MaxFunc naming would be better in my opinion. what do you think @ReneWerner87 @gaby @ReneWerner87 @sixcolors

Reply:
yeah
@luk3skyw4lker can you change this

Reply:
@luk3skyw4lker

Reply:
@ReneWerner87 @efectn done guys, sorry to take so long